pkg/api/targets_test.go

```diff
@@ -291,3 +298,77 @@ func TestGetTargetMTTR(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateTarget(t *testing.T) {
+	newTarget := model.Target{
+		Identifier: "example.com",
+		Tags:       []string{"mytag"},
+	}
+
+	tcs := []test.Case{
```

Comment:
I think it would be good to include one more test case in which the `Identifier` is valid and `Tags` is empty. It should return a `StatusCreated`, right?

Also, what would happen if the user provides an empty `Identifier`?

Reply:
Wait... not I have realised that we are not testing the method `(api *API) CreateTarget`, right?

Reply:
I added the case for the payload with valid `identifier` and no tags.
The case for void identifier was already there, but I changed to name to it to make it more clear, as the naming was referring more on the expected result than the input.